src/order/bounded_order.lean

```diff
@@ -294,65 +259,26 @@ end semilattice_inf_bot
 
 /-! ### Bounded lattice -/
 
-/-- A bounded lattice is a lattice with a top and bottom element,
+-- TODO: rename `bounded_order` since it no longer requires `lattice`
+/-- A bounded order describes an order `(≤)` with a top and bottom element,
   denoted `⊤` and `⊥` respectively. This allows for the interpretation
   of all finite suprema and infima, taking `inf ∅ = ⊤` and `sup ∅ = ⊥`. -/
-class bounded_lattice (α : Type u) extends lattice α, has_top α, has_bot α :=
-(le_top : ∀ a : α, a ≤ ⊤)
-(bot_le : ∀ a : α, ⊥ ≤ a)
-
-@[priority 100] -- see Note [lower instance priority]
-instance semilattice_inf_top_of_bounded_lattice (α : Type u) [bl : bounded_lattice α] :
-  semilattice_inf_top α :=
-{ ..bl }
-
-@[priority 100] -- see Note [lower instance priority]
-instance semilattice_inf_bot_of_bounded_lattice (α : Type u) [bl : bounded_lattice α] :
-  semilattice_inf_bot α :=
-{ ..bl }
-
-@[priority 100] -- see Note [lower instance priority]
-instance semilattice_sup_top_of_bounded_lattice (α : Type u) [bl : bounded_lattice α] :
-  semilattice_sup_top α :=
-{ ..bl }
-
-@[priority 100] -- see Note [lower instance priority]
-instance semilattice_sup_bot_of_bounded_lattice (α : Type u) [bl : bounded_lattice α] :
-  semilattice_sup_bot α :=
-{ ..bl }
-
-theorem bounded_lattice.ext {α} {A B : bounded_lattice α}
```

Comment:
Do we have a replacement for this?

Reply:
I guess we should keep this around. But the proof can be made much shorter (2-3 lines?) using `order_top.ext` and `order_bot.ext`.